Jenkinsfile

```diff
@@ -61,6 +62,20 @@ pipeline {
             }
         }
 
+        stage('Linux test') {
+            when { expression { env.ghprbPullId } }
```

Comment:
This stage is going to be executed for all repositories from GitHub, which is not necessary (including https://github.com/Juniper/contrail-windows).

Reply:
The goal was just to make gerrit logs cleaner and don't let random people wonder "why are linux tests running in windows CI". I don't think `contrail-windows` would be a problem here.